docs/manual/manual_velocity_control_mechanisms.rst

```diff
@@ -0,0 +1,419 @@
+
+=========================================
+Velocity and Voltage Control Mechanisms
+=========================================
+Control Velocity
++++++++++++++++++++++
+Velocity Control Through :ref:`Propeller Motor Controller <propeller_motor_control>` v. :ref:`Multi Turn Angle Controller <multi_turn_control_label>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+All *Control Velocity* commands use closed loop PID control in order to control either a target velocity or position. 
+The controllers available on each module are firmware dependent. In order to find what controllers are available with your 
+module and firmware, please see your module's page available on the left-hand side of this page.
+
+Our :ref:`Propeller Motor Controller <propeller_motor_control>`, available only while using speed firmware, uses a velocity PID controller in which the input target is 
+a velocity, and error is calculated on the difference between the module's actual velocity and the target. 
+This is the default controller used when the module receives commands through :ref:`Analog Hobby Protocols <manual_hobby>` or 
+:ref:`DroneCAN <manual_dronecan>`. An important note is that while Hobby and DroneCAN commands are passed into the Propeller Motor Controller, the 
+exact type of control depends on the configured *mode* parameter. Please refer to these linked pages in order to learn more about  
+Hobby and DroneCAN, and the *mode* parameter.
+
+.. figure:: ../_static/manual_images/velo_voltage_control/prop_motor_controller.png
+    :align: center
+    :scale: 90%
+    :alt: Propeller Motor Controller PID Loop
+
+    Propeller Motor Controller PID Loop
+
+Our :ref:`Multi Turn Angle Controller <multi_turn_control_label>` (available on all servo firmware and on select speed firmware) is generally meant for angle based control, 
+and is detailed more in our :ref:`Angle Control Mechanisms documentation <manual_angle_control_mechanisms>`. It can, however, also control the module's velocity, 
+with an important note that it always controls based on a target position. This means that when sent a velocity command, the *Multi Turn Angle Controller* 
+advances the displacement angle target at the commanded velocity.
+
+.. figure:: ../_static/manual_images/velo_voltage_control/multi_turn_controller.png
+    :align: center
+    :scale: 90%
+    :alt: Multi Turn Motor Controller PID Loop
+
+    Multi Turn Motor Controller PID Loop
+
+Due to each PID controller's nature, each reacts differently to changes in the proportional, 
+integral, and derivative gains. An important distinction is that even when controlling the *Multi Turn Angle Controller* with a velocity, 
+the proportional gain relates to a positional error, the integral gain to the positional error times time, and the derivative 
```

Comment:
Sort of a subjective thing, but I don't like saying "times time". "multiplied by time" or "the product of positional error and time" or something along those lines would be better in my opinion